loc/templates/page_text.html

```diff
@@ -38,11 +38,13 @@
     </div>
 
     <div>
-    {{page.ocr.text|escape|linebreaks}}
+    {% if text %}
+    {{ text.0|linebreaks }}
```

Comment:
Same question about whether this can possibly have more than one element

Reply:
added join filter to the template